lib/ddtrace/ext/debug.rb

```diff
@@ -0,0 +1,27 @@
+module Datadog
+  module Ext
+    module Debug
+      # Health
+      module Health
+        # Metrics
+        module Metrics
+          ENV_ENABLED = 'DD_DEBUG_HEALTH_METRICS_ENABLED'.freeze
+
+          METRIC_API_ERRORS = 'datadog.tracer.api.errors'.freeze
+          METRIC_API_REQUESTS = 'datadog.tracer.api.requests'.freeze
+          METRIC_API_RESPONSES = 'datadog.tracer.api.responses'.freeze
+          METRIC_QUEUE_ACCEPTED = 'datadog.tracer.queue.accepted'.freeze
+          METRIC_QUEUE_ACCEPTED_LENGTHS = 'datadog.tracer.queue.accepted_lengths'.freeze
+          METRIC_QUEUE_ACCEPTED_SIZE = 'datadog.tracer.queue.accepted_size'.freeze
```

Comment:
For these two sub-metrics of `datadog.tracer.queue.accepted`, would it make sense to have the metric name separator like so: `datadog.tracer.queue.accepted.lengths` `datadog.tracer.queue.accepted.size`?

Reply:
Yeah I think that could make sense; something we'd want to reconcile with our standards though.